contributors/design-proposals/propagation.md

```diff
@@ -73,13 +79,18 @@ type VolumeMount struct {
 	// Required.
 	MountPath string `json:"mountPath"`
 	// Optional.
-	Propagation string `json:"propagation"`
+	Propagation PropagationMode `json:"propagation,omitempty"`
 }
 ```
 
+Default would be `RSlave`, which should not break backward compatibility,
+`RShared` must be explicitly requested.
+
 Opinion against this:
 
-1. This will affect all volumes, while only HostPath need this.
+1. This will affect all volumes, while only HostPath need this. It could be
```

Comment:
This is an interesting use case in its own, and if we want to consider emptyDir with propagation, we need to pay extra attention to cleanup.  I have observed that this can lead to "leaked" mounts.  It definitely changes the calculus on this approach.

What is namedEmptyDir ?